pages/multiplayerGame.js

```diff
@@ -186,19 +186,15 @@ export default class MultiplayerGame extends Component {
                     <Section>
                         <Bloc title="Waiting for connection" isLoading="true">
                             <p>This page is only accessile online.</p>
-                        </Bloc>
-                    </Section>
-                    <Section>
-                        <Row>
-                            <div className="buttons-wrapper">
+                            <Col>
```

Comment:
I would rather rename it to `<Column />`. Generally, we prevent abbreviations, except for well documented ubiquitous language.

Reply:
I agree with you in most situations. But here I think `Col` is the best choice because my app is based on the materialize css framework that use `col` instead of `column`.